interfaces/cython/cantera/examples/onedim/ion_flame.py

```diff
@@ -19,20 +19,16 @@
 
 # Set up flame object
 f = ct.IonFlame(gas, width=width)
-f.set_refine_criteria(ratio=3, slope=0.06, curve=0.12)
+f.set_refine_criteria(ratio=3, slope=0.05, curve=0.1)
 f.show_solution()
 
 # stage one
 f.solve(loglevel=loglevel, auto=True)
 
-# stage two
-f.solve(loglevel=loglevel, stage=2, enable_energy=False)
-f.solve(loglevel=loglevel, stage=2, enable_energy=True)
-
-# stage three
+# stage three (stage two is deprecated)
 f.solve(loglevel=loglevel, stage=3, enable_energy=True)
```

Comment:
Given the change to `IonFlame.solve`, shouldn't this just be `stage=2` now? With the current call to `stage=3`, I think this ends up doing nothing. So, perhaps `IonFlame.solve` should raise an exception if `stage` is not either 1 or 2.